libs/ts-rest/core/src/lib/dsl.ts

```diff
@@ -186,6 +192,8 @@ const recursivelyApplyOptions = <T extends AppRouter>(
           {
             ...value,
             headers: zodMerge(options?.baseHeaders, value.headers),
+            strictStatusCodes:
+              options?.strictStatusCodes ?? value.strictStatusCodes,
```

Comment:
Same as the other comment
```suggestion
            strictStatusCodes:
              value.strictStatusCodes ?? options?.strictStatusCodes,
```